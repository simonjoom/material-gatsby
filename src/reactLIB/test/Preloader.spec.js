import React from 'react';
import { shallow } from 'enzyme';
import Preloader from '../src/Preloader';

describe('<Preloader />', () => {
  test('should render a Preloader', () => {
    const wrapper = shallow(
      <Preloader
        className="find"
        size="big"
        active={false}
        color="red"
        flashing
      />
    );

    expect(wrapper).toMatchSnapshot();
  });

  test('should render a Preloader with default properties', () => {
    const wrapper = shallow(<Preloader />);

    expect(wrapper).toMatchSnapshot();
  });
});
