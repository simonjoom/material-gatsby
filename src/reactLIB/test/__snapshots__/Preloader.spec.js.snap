// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloader /> should render a Preloader 1`] = `
<div
  className="find preloader-wrapper size"
>
  <Spinner
    color="blue"
    key="blue"
    only={false}
  />
  <Spinner
    color="red"
    key="red"
    only={false}
  />
  <Spinner
    color="yellow"
    key="yellow"
    only={false}
  />
  <Spinner
    color="green"
    key="green"
    only={false}
  />
</div>
`;

exports[`<Preloader /> should render a Preloader with default properties 1`] = `
<div
  className="preloader-wrapper active"
>
  <Spinner
    color="blue"
    only={true}
  />
</div>
`;
